### PR TITLE
fix: update CI to use LLVM-19

### PR DESCRIPTION
**Pull request by varex83**

### CODE REVIEW

File: scripts/setup_native_deps.sh [hunk 1]
```diff
@@ -16,17 +16,17 @@ setup_llvm_deps() {
...
 	Darwin)
 		brew update
-		brew install llvm@18
+		brew install llvm@19
 
 		LIBRARY_PATH=/opt/homebrew/lib
-		MLIR_SYS_180_PREFIX="$(brew --prefix llvm@18)"
-		LLVM_SYS_181_PREFIX="$MLIR_SYS_180_PREFIX"
-		TABLEGEN_180_PREFIX="$MLIR_SYS_180_PREFIX"
+		MLIR_SYS_190_PREFIX="$(brew --prefix llvm@18)"
```
Comment (xrvdg) [inline scripts/setup_native_deps.sh]: This should be `@19`. 
```suggestion
		MLIR_SYS_190_PREFIX="$(brew --prefix llvm@19)"
```

File: scripts/setup_native_deps.sh [hunk 2]
```diff
@@ -16,17 +16,17 @@ setup_llvm_deps() {
...
+		LLVM_SYS_190_PREFIX="$MLIR_SYS_190_PREFIX"
```
Comment (rodrodros) [inline scripts/setup_native_deps.sh]: `LLVM_SYS_191_PREFIX` (190 to 191)

File: scripts/setup_native_deps.sh [hunk 3]
```diff
@@ -16,17 +16,17 @@ setup_llvm_deps() {
...
+		TABLEGEN_190_PREFIX="$MLIR_SYS_190_PREFIX"
 
 		export LIBRARY_PATH
-		export MLIR_SYS_180_PREFIX
-		export LLVM_SYS_181_PREFIX
-		export TABLEGEN_180_PREFIX
+		export MLIR_SYS_190_PREFIX
+		export LLVM_SYS_190_PREFIX
```
Comment (rodrodros) [inline scripts/setup_native_deps.sh]: 190 to 191 on `LLVM_SYS_190_PREFIX`

File: scripts/setup_native_deps.sh [hunk 4]
```diff
@@ -35,22 +35,22 @@ setup_llvm_deps() {
...
+		MLIR_SYS_190_PREFIX=/usr/lib/llvm-19/
+		LLVM_SYS_190_PREFIX=/usr/lib/llvm-19/
+		TABLEGEN_190_PREFIX=/usr/lib/llvm-19/
 
-		export MLIR_SYS_180_PREFIX
-		export LLVM_SYS_181_PREFIX
-		export TABLEGEN_180_PREFIX
+		export MLIR_SYS_190_PREFIX
+		export LLVM_SYS_190_PREFIX
+		export TABLEGEN_190_PREFIX
```
Comment (rodrodros) [inline scripts/setup_native_deps.sh]: Some thing with named var here

File: scripts/setup_native_deps.sh [hunk 5]
```diff
@@ -60,9 +60,9 @@ setup_llvm_deps() {
 
 	# GitHub Actions specific
 	[ -n "$GITHUB_ACTIONS" ] && {
-    echo "MLIR_SYS_180_PREFIX=$MLIR_SYS_180_PREFIX" >> $GITHUB_ENV
-    echo "LLVM_SYS_181_PREFIX=$LLVM_SYS_181_PREFIX" >> $GITHUB_ENV
-    echo "TABLEGEN_180_PREFIX=$TABLEGEN_180_PREFIX" >> $GITHUB_ENV
+    echo "MLIR_SYS_190_PREFIX=$MLIR_SYS_190_PREFIX" >> $GITHUB_ENV
+    echo "LLVM_SYS_190_PREFIX=$LLVM_SYS_190_PREFIX" >> $GITHUB_ENV
+    echo "TABLEGEN_190_PREFIX=$TABLEGEN_190_PREFIX" >> $GITHUB_ENV
```
Comment (rodrodros) [inline scripts/setup_native_deps.sh]: And here

File: .github/actions/setup-native-deps/action.yml [hunk 6]
```diff
@@ -23,6 +23,6 @@ runs:
       id: set-env-vars
       shell: bash
       run: |
-        echo "MLIR_SYS_180_PREFIX=/usr/lib/llvm-18/" >> $GITHUB_ENV &&
-        echo "LLVM_SYS_181_PREFIX=/usr/lib/llvm-18/" >> $GITHUB_ENV &&
-        echo "TABLEGEN_180_PREFIX=/usr/lib/llvm-18/" >> $GITHUB_ENV
\ No newline at end of file
+        echo "MLIR_SYS_190_PREFIX=/usr/lib/llvm-19/" >> $GITHUB_ENV &&
+        echo "LLVM_SYS_190_PREFIX=/usr/lib/llvm-19/" >> $GITHUB_ENV &&
```
Comment (rodrodros) [inline .github/actions/setup-native-deps/action.yml]: I think this is the last place

**Pull request merged**